client/my-sites/checkout/composite-checkout/payment-method-helpers.js

```diff
@@ -156,6 +157,15 @@ export async function submitStripeRedirectTransaction( paymentMethodId, transact
 	return submit( formattedTransactionData );
 }
 
+export async function submitEbanxTefTransaction( transactionData, submit ) {
```

Comment:
I don't think this function is used anywhere, right?

Reply:
I rebased at some point and this got included in the rebased branch by mistake- it's gone now.

Reply:
But... it's still here. 🤔

Reply:
Ah- I see what I did wrong. I `rg`d for the function name to find all the instances of it, but misspelled the name so there were no results. Meanwhile I really did have to remove some rebase gunk and assumed this was that function. Thanks for checking again :)